### PR TITLE
created bio

### PR DESCRIPTION
**Pull request by SWAPNACHEMBOTH**
### General Checks

- [x] the branch is up to date with `main`/`master`
- [x] the code works when pulled and run locally
- [x] All CI checks pass
- [x] all conflicts are resolved (if any)
- [x] PR has a descriptive title
- [x] PR has appropriate labels and milestones for easy identification
- [x] PR it is assigned to the owner
- [x] reviewers are assigned
- [x] the PR contributes only one focused change
- [x] It is in the appropriate column in the project board (if necessary)
- [x] has short and clear description
- [x] is linked to an issue (if it is related)
- [ ] feedback is addressed (if any and if it is appropriate feedback.)

## Markdown

- [x] the markdown source is formatted
- [x] spelling and grammar is correct in all text
- [x] The markdown looks correct when you preview the file
- [ ] all links and images work

### CODE REVIEW

**Comment by Alexander-Segovia** [comment]
Hi @SWAPNACHEMBOTH There is a lot of HTML in your MD file, try using exclusively Markdown for this task and upload the changes :)

File: people/swapnabio.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h1 align="center">Hi 👋, I'm Swapna Chemboth. Currently living in Brussels,Belgium</h1>
```
Comment (prgrmfl) [inline people/swapnabio.md:1]: Hi @SWAPNACHEMBOTH 
You can replace all the h1, h2.. with markdown equivalents such as '# heading' for h1

File: people/swapna.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Hi, this is Swapna
```
Comment (prgrmfl) [inline people/swapna.md:1]: Maybe you can consider deleting this file, as there are two bio files with your name.

**Review by prgrmfl — CHANGES_REQUESTED** [review]
Hi @SWAPNACHEMBOTH 
1. You can do `npm install` and then do linting checks from your terminal/CLI.
2. - You should do `git checkout swapna` from your terminal, to go to your branch
    - Then use `git pull` command to update the master branch  
    - And use `git pull origin master` to update your branch-swapna with the latest changes in master branch  
    - make changes if any in your swapnabio.md file
    - `git add /people/swapnabio.md` to add your file and add other files if there are any changes you can also use `git add .` to add all the changed files
    - `git commit -m "commit message"` to commit the changes you added
    - `git push` to push the committed files to your branch in github

**Pull request merged**